### PR TITLE
[TG-210] [feat] made scenarios update their status when they end working

### PR DESCRIPTION
**Pull request by quasiunafantasia**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-backend/pull/38" target="_blank">ReviewNinja

File: src/scenarios/scenario-runner/runner.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ const scenarioApiManager = require('./scenario-api.manager');
 const inputStream = new Rx.Subject();
 const outputStream = new Rx.Subject();
 
+//0 comand arg is node, 1 is path to file
+const argvScenarioBodyIndex = 2;
```
Comment (kosich) [inline src/scenarios/scenario-runner/runner.js]: use UPPER_CASE

File: src/scenarios/scenario-runner/scenario-api.manager.js [hunk 2]
```diff
@@ -103,6 +105,13 @@ function create(inputStream, outputStream){
             };
 
             function onHandler(eventName, devices, cb){
+                if(!hasDevicesSubscribed && devices.length) {
+                    process.on('message', (message) => {
```
Comment (kosich) [inline src/scenarios/scenario-runner/scenario-api.manager.js]: dont like it here

**Comment by kosich** [comment]
⭐

File: src/scenarios/scenario-runner/scenario-api.manager.js (hunk 2 above)
Comment (quasiunafantasia) [inline src/scenarios/scenario-runner/scenario-api.manager.js]: neither do i :(

**Pull request merged**